### PR TITLE
ADEN-2710 Disable Taboola right rail module assertions

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTaboolaOasis.java [hunk 1]
```diff
@@ -19,12 +19,12 @@ public void adsTaboolaOasis() {
     AdsTaboolaObject adsTaboolaObject = new AdsTaboolaObject(driver);
     adsTaboolaObject.getUrl(testedPage);
 
-    adsTaboolaObject.verifyTaboolaContainer(AdsTaboolaObject.RIGHT_RAIL_CSS_SELECTOR);
     adsTaboolaObject.verifyTaboolaContainer(AdsTaboolaObject.BELOW_ARTICLE_CSS_SELECTOR);
-
-    adsTaboolaObject.verifyTaboolaAdsPresent(AdsTaboolaObject.RIGHT_RAIL_CSS_SELECTOR);
     adsTaboolaObject.verifyTaboolaAdsPresent(AdsTaboolaObject.BELOW_ARTICLE_CSS_SELECTOR);
 
+//    @TODO Assertions based on configuration: ADEN-2729
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTaboolaOasis.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTaboolaOasis.java [hunk 2]
```diff
@@ -19,12 +19,12 @@ public void adsTaboolaOasis() {
...
+//    adsTaboolaObject.verifyTaboolaContainer(AdsTaboolaObject.RIGHT_RAIL_CSS_SELECTOR);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTaboolaOasis.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by nandy-andy** [comment]
I've created a follow-up ticket: [ADEN-2731](https://wikia-inc.atlassian.net/browse/ADEN-2731). And with the ticket the code changes LGTM.

**Comment by PiotrGackowski** [comment]
LGTM

**Pull request merged**